src/bench/nanobench.h

```diff
@@ -1905,7 +1905,7 @@ PerformanceCounters& performanceCounters() {
 // Windows version of doNotOptimizeAway
 // see https://github.com/google/benchmark/blob/master/include/benchmark/benchmark.h#L307
 // see https://github.com/facebook/folly/blob/master/folly/Benchmark.h#L280
-// see https://docs.microsoft.com/en-us/cpp/preprocessor/optimize
+// see https://learn.microsoft.com/en-us/cpp/preprocessor/optimize?view=msvc-170
```

Comment:
No need to specify the exact VS version. This will point at the latest one:
```suggestion
// see https://learn.microsoft.com/en-us/cpp/preprocessor/optimize
```

Reply:
✔️ Removed exact VS version in `src/bench/nanobench.h`.